src/frontend/apps/desk/src/features/auth/api/types.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Represents user retrieved from the API.
+ * This interface is incomplete, and will be
+ * refactored in a near future.
```

Comment:
nit: should we consider it complete and update the docstring?

Reply:
https://github.com/numerique-gouv/people/pull/73/commits/19ac4929c997828bacfa7438f195278f1eba469d#diff-438788777e79b5da4a9993acecd47c6048e9fbb5ba807e1d53408f7b263e8687R1-R7